### DISCUSSION TITLE
GLPI Inventory Plugin "not have sufficient rights for this action"

### DISCUSSION THREAD

**Discussion by martinautox** [glpi-project/glpi-agent#376] [Q&A] [answered]
Hello all,

I hope that I can be helped here. I am in the process of installing a new server with the current GLPI v10.0.7 in addition to my current Live GLPI version, which has also worked so far.
The base operating system is RHEL 8.

I installed the GLPI Inventory Plugin in version 1.2.1

On the server side I can configure everything. I also installed the GLPI agent on the client and the GLPI server was specified on the server (https://xxxxx.com/plugins/glpiinventory).

However, I always get the error message "No answer from server at https://......" in the logs.

If I then call the link https://xxxxxx.com/plugins/glpiinventory in the browser I always get the following message: you do not have sufficient rights for this action.

In my productive live environment it works without problems. What is the reason for this?

**Comment by cedric-anne** [comment]
> However, I always get the error message "No answer from server at https://......" in the logs.

Are you sure your agent is able to do requests to the server?

> If I then call the link https://xxxxxx.com/plugins/glpiinventory in the browser I always get the following message: you do not have sufficient rights for this action.

You should verify the profile you are using. Maybe it has no access rights on `GLPI Inventory` features.

**Reply by martinautox** [reply]
> Are you sure your agent is able to do requests to the server?

Yes my agent is allowed to make requests to the server, it works on the live productive server.
But I would be grateful if I could get some information where I can check this.


> You should verify the profile you are using. Maybe it has no access rights on GLPI Inventory features.

The profile (whichever one is used) has all rights according to the GLPI server.

**Comment by martinautox** [comment, marked as answer]
I just found the solution myself.
There is a post Issue#13213

There it is pointed out that in the new versions the inventory function is deactivated from start and you have to activate it manually.
